### PR TITLE
resolve possible null pointer exceptions

### PR DESCRIPTION
**Pull request by zkzong**
resolve possible null pointer exceptions

### CODE REVIEW

**Comment by biyuhao** [comment]
Hi, please leave the line ending characters (LF) unchanged.

File: dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/wrapper/MockClusterInvoker.java [hunk 1]
```diff
@@ -1,167 +1,167 @@
...
+    public Class<T> getInterface() {
+        return directory.getInterface();
+    }
+
+    @Override
+    public Result invoke(Invocation invocation) throws RpcException {
+        Result result = null;
+
+        String value = directory.getUrl().getMethodParameter(invocation.getMethodName(), Constants.MOCK_KEY, Boolean.FALSE.toString()).trim();
+        if (value.length() == 0 || "false".equalsIgnoreCase(value)) {
```
Comment (Leishunyu) [inline dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/wrapper/MockClusterInvoker.java]: I think it is better to define a constant here.

**Comment by Moriadry** [comment]
try   `git config core.autocrlf false`  to avoid autocrlf.

**Comment by ralf0131** [comment]
Hi, sorry but could you please resolve the conflict first?

File: dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/wrapper/MockClusterInvoker.java [hunk 2]
```diff
@@ -1,169 +1,169 @@
...
-        }
-        return invokers;
-    }
-
-    @Override
-    public String toString() {
-        return "invoker :" + this.invoker + ",directory: " + this.directory;
-    }
-}
+/*
```
Comment (AlbumenJ) [inline dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/wrapper/MockClusterInvoker.java:1]: please leave file endline format not changed

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

**Comment by AlbumenJ** [comment]
Close for long time no response.

Please feel free to reopen if you have any question.

If you think these changes still useful in the latest master branch, please submit a new pull request.

**Pull request closed without merging**